lib/NetworkManager.js

```diff
@@ -188,7 +188,7 @@ class NetworkManager extends EventEmitter {
    * @param {?Object} securityDetails
    */
   _handleRequestRedirect(request, redirectStatus, redirectHeaders, fromDiskCache, fromServiceWorker, securityDetails) {
-    const response = new Response(this._client, request, redirectStatus, redirectHeaders, fromDiskCache, fromServiceWorker, securityDetails);
+    const response = new Response(this._client, request, redirectStatus, redirectHeaders, fromDiskCache, fromServiceWorker, securityDetails, null, null);
```

Comment:
let's plumb the remoteIDAddress and remotePort to the redirect responses (also let's cover this with a test)

Reply:
Good point - I'll update with a test later today.